test/parseLdapFilter.tests.js

```diff
@@ -0,0 +1,53 @@
+const ldap = require("ldapjs");
+const expect = require("chai").expect;
+const parseLdapFilter = require("../lib/parseLdapFilter");
+
+describe("parseLdapFilter", () => {
```

Comment:
Thanks for doing the change. Could you also add a test case for functions in lib/users.js to validate return value in case of invalid filter. Atleast for getByUserName. Thanks!

Reply:
@vmarceau Sorry, if I missed. Where is this mocked ldap server used in the test?